drivers/fuel_gauge/sbs_gauge/sbs_gauge.c

```diff
@@ -307,17 +331,39 @@ static const struct fuel_gauge_driver_api sbs_gauge_driver_api = {
 	.get_property = &sbs_gauge_get_props,
 	.set_property = &sbs_gauge_set_props,
 	.get_buffer_property = &sbs_gauge_get_buffer_prop,
+#ifdef SBS_GAUGE_CUTOFF_SUPPORT
+	.battery_cutoff = &sbs_gauge_do_battery_cutoff,
+#endif
 };
 
-/* FIXME: fix init priority */
+#ifdef SBS_GAUGE_CUTOFF_SUPPORT
+#define SBS_GAUGE_CONFIG_INIT(index)                                                               \
+	.cutoff_support = DT_PROP_OR(DT_DRV_INST(index), battery_cutoff_support, false),           \
+	.cutoff_reg_addr = DT_PROP_OR(DT_DRV_INST(index), battery_cutoff_reg_addr, 0),             \
+	.cutoff_payload = DT_PROP_OR(DT_DRV_INST(index), battery_cutoff_payload, {}),
+#else
+#define SBS_GAUGE_CONFIG_INIT(index) \
```

Comment:
Unnecessary line continuation.
```suggestion
#define SBS_GAUGE_CONFIG_INIT(index)
```